### PR TITLE
added rg as option for ack command

### PR DESCRIPTION
**Pull request by ssnnoo**
add skeleton for file search via rg; rg command should be customized to rg --color always by user

### CODE REVIEW

**Comment by ssnnoo** [comment]
actually better would be to set "rg -n -H --no-heading --color always -e " as ack command by the user. Maybe this could be added to the documentation?

**Comment by joaotavora** [comment]
You can change the pull request to reflect what you think is the better default, and I will merge it.

File: ack.el [hunk 1]
```diff
@@ -314,8 +315,8 @@ This gets tacked on the end of the generated expressions.")
   (interactive)
   (delete-minibuffer-contents)
   (let ((ack (or (car (split-string ack-command nil t)) "ack")))
-    (if (equal ack "ag")
-        (skeleton-insert `(nil ,ack " -ig '" _ "'"))
+    (cond ((equal ack "ag") (skeleton-insert `(nil ,ack " -ig '" _ "'")))
+	  ((equal ack "rg") (skeleton-insert `(nil ,ack " --color always --files --iglob '*" _ "*'")))
```
Comment (joaotavora) [inline ack.el:319]: Maybe you could break this very long line?

Comment (ssnnoo) [reply ack.el]: ok, hope it looks better now

**Comment by ssnnoo** [comment]
I modified the default and shortened the lines.

**Comment by ssnnoo** [comment]
added -S option to make ripgrep behave more like ag regarding case sensitivity and fixed typo

**Comment by joaotavora** [comment]
I merged this manually.

**Pull request closed without merging**